src/spec/ToastContainer.spec.jsx

```diff
@@ -0,0 +1,70 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import { describe, it, expect, vi } from "vitest";
+
+import ToastContainer from "../components/Toast/ToastContainer";
+import { act } from "react";
```

Comment:
```suggestion
import { act } from "react";
import { render, screen, fireEvent } from "@testing-library/react";
import { describe, it, expect, vi } from "vitest";

import ToastContainer from "../components/Toast/ToastContainer";
```
라이브러리와 컴포넌트를 구분하면 좋을 것 같습니다.

Reply:
수정했습니다!